### PR TITLE
A0-1459: Modify workflow to push and deploy the image

### PR DESCRIPTION
**Pull request by mikogs**

### CODE REVIEW

**Review by Kosiare — APPROVED** [review]

File: .github/actions/deploy-to-devnet/action.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      uses: EndBug/add-and-commit@v5.1.0
+      env:
+        GITHUB_TOKEN: ${{ inputs.github_token }}
+      with:
+        author_name: AlephZero Automation
+        author_email: alephzero@10clouds.com
+        message: 'Updating Indexer ${{ inputs.registry }} docker image tag: ${{ inputs.image_tag }}'
+        add: '*.yaml'
+        cwd: 'aleph-apps'
+        branch: A0-1459-indexer
```
Comment (Kosiare) [inline .github/actions/deploy-to-devnet/action.yaml:97]: Change to `main` before merge.

File: .github/actions/deploy-to-devnet/action.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        echo "branch=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
+        echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+    - name: GIT | Checkout aleph-apps repo
+      uses: actions/checkout@master
+      with:
+        repository: Cardinal-Cryptography/aleph-apps
+        token: ${{ inputs.github_token }}
+        path: "aleph-apps"
+        ref: A0-1459-indexer
```
Comment (Kosiare) [inline .github/actions/deploy-to-devnet/action.yaml:48]: Change to `main` before merge.

File: .github/actions/deploy-to-devnet/action.yaml (hunk 1 above)
Comment (mikogs) [reply .github/actions/deploy-to-devnet/action.yaml]: changed

File: .github/actions/deploy-to-devnet/action.yaml (hunk 2 above)
Comment (mikogs) [reply .github/actions/deploy-to-devnet/action.yaml]: changed

**Pull request merged**